File: 3131-find-the-integer-added-to-array-i/3131-find-the-integer-added-to-array-i.cpp
class Solution {
public:
    int addedInteger(vector<int>& nums1, vector<int>& nums2) {
        int max1 = *max_element(nums1.begin(), nums1.end());       
        int max2 = *max_element(nums2.begin(), nums2.end());  
        
        int ans = max2-max1;

        // if (min1 > min2)  ansc = -ansc;
        
        return ans;
    }
};

File: 3131-find-the-integer-added-to-array-i/3131-find-the-integer-added-to-array-i.java
class Solution {
    public int addedInteger(int[] nums1, int[] nums2) {
        int nums1Total = 0, nums2Total = 0;
        for(int i=0; i<nums1.length; i++) {
            nums1Total += nums1[i];
            nums2Total += nums2[i];
        }

        return (nums2Total - nums1Total)/nums1.length;
    }
}